targets/XPlatCppSdk/source/cppsdk/XPlatCppLinux.vcxproj.filters.ejs

```diff
@@ -48,10 +54,32 @@
     <ClInclude Include="include\playfab\PlayFabPluginManager.h">
       <Filter>Header Files\playfab</Filter>
     </ClInclude>
+    <ClInclude Include="include\playfab\QoS\DataCenterResult.h">
+      <Filter>Header Files\playfab\QoS</Filter>
+    </ClInclude>
+    <ClInclude Include="include\playfab\QoS\PingResult.h">
+      <Filter>Header Files\playfab\QoS</Filter>
+    </ClInclude>
+    <ClInclude Include="include\playfab\QoS\PlayFabQoSApi.h">
+      <Filter>Header Files\playfab\QoS</Filter>
+    </ClInclude>
+    <ClInclude Include="include\playfab\QoS\QoS.h">
+      <Filter>Header Files\playfab\QoS</Filter>
+    </ClInclude>
+    <ClInclude Include="include\playfab\QoS\QoSResult.h">
+      <Filter>Header Files\playfab\QoS</Filter>
+    </ClInclude>
+    <ClInclude Include="include\playfab\QoS\QoSSocket.h">
+      <Filter>Header Files\playfab\QoS</Filter>
+    </ClInclude>
+    <ClInclude Include="include\playfab\QoS\XPlatSocket.h">
+      <Filter>Header Files\playfab\QoS</Filter>
+    </ClInclude>
+    <ClInclude Include="source\playfab\PlayFabAdminApi.cpp">
   </ItemGroup>
   <ItemGroup>
     <ClCompile Include="stdafx.cpp">
-      <Filter>Source Files</Filter>
+      <Filter>Source Files\playfab</Filter>
```

Comment:
>playfab [](start = 27, length = 7)

stdafx.cpp is not in "playfab" subfolder, this is not necessary. Consider removing this particular modification

Reply:
agree.